src/Angor/Avalonia/Angor.Model.Implementation/DependencyFactory.cs

```diff
@@ -0,0 +1,37 @@
+using Angor.Shared.Services;
+using Microsoft.Extensions.Logging;
+using Nostr.Client.Client;
+
+namespace Angor.Model.Implementation;
+
+public static class DependencyFactory
+{
+    public static IRelayService GetRelayService(ILoggerFactory loggerFactory)
```

Comment:
just wondering why not use the standard dependency injection?

Reply:
Good question! I chose this approach because the number of dependencies is still relatively small and adding a DI container felt like overengineering at this stage. However, if you think we should switch to standard DI, I can prepare a PR with the changes.

Reply:
I am not sure what DI you had in mind to use but the default on the web code is `Microsoft.Extensions.DependencyInjection.ServiceProvider`

![image](https://github.com/user-attachments/assets/a14898f0-5693-4772-a4e0-d0620fc50f94)

So probably we just use the same one

Reply:
I tried to use use of Microsoft.Extensions.DependencyInjection. It's done and it works perfectly, but I think that change should be done in a follow-up PR. It's a bit of scope and I'd like to keep it separated. What do you think?